eng/scripts/Language-Settings.ps1

```diff
@@ -103,7 +103,7 @@ function Get-AllPackageInfoFromRepo($serviceDirectory)
   if ($serviceDirectory) {
     $searchPath = Join-Path $searchPath $serviceDirectory
     # If we have a service directory only include the immediate match, no nested go.mod files
-    [array]$pkgFiles = Get-ChildItem -Path $searchPath -Include "go.mod" -Depth 1
+    [array]$pkgFiles = Get-ChildItem -Path $searchPath -Include "go.mod" -Depth 0
```

Comment:
Why is the depth even needed given it isn't using recursive.

Reply:
I guess it's a holdover from using `-Include` which only works with `-Depth`. I should change this whole command to just `Get-ChildItem "$searchPath/go.mod"`.